README.md

```diff
@@ -28,6 +28,31 @@ If you have any questions or need help getting started, please reach out though
 | [Google Season of Docs](https://developers.google.com/season-of-docs)             | Mentoring initiative for the technical writers                                                                                            | [seasonofdocs](/programs/seasonofdocs#readme)     |
 | [Outreachy](https://www.outreachy.org)                                            | Mentoring initiative for the communities traditionally underrepresented in tech                                                           | [outreachy](/programs/outreachy#readme)           |
 
+## Program Statistics
+
+These are the number of successful internships per year for each program.
+
+| Year | Program                          | Internships | Total (per year) |
+|------|----------------------------------|-------------|------------------|
+| 2023 | LFX Mentorship                   | TBD         | TBD              |
+|      | GSoC                             | TBD         |                  |
+| 2022 | LFX Mentorship                   | TBD         | TBD              |
+|      | GSoC                             | TBD         |                  |
```

Comment:
@nate-double-u 
I see GSoC 2022 has 16 successful projects: https://summerofcode.withgoogle.com/archive/2022/organizations/cncf

Can you find out the number for LFX? I couldn't find such info publicly.

Reply:
I just move the stats from a file to the README. We should propable open an issue to update the 2022 stats.

Reply:
I think https://github.com/cncf/mentoring/pull/1012 takes care of this.